spec/std/process_spec.cr

```diff
@@ -81,6 +103,21 @@ describe Process do
     $?.exit_code.should eq(0)
   end
 
+  it "can chroot" do
+    status, output = build_and_run <<-CODE
+      Process.chroot("/var/empty")
```

Comment:
You need to ensure this actually exists. I would prefer to have it either in the project folder or `/tmp` (we need to resolve #5811), but not `/var`.

Reply:
`/var/empty` is a standard location on UNIX filesystems. It is used almost exclusively for chroming processes which don't need file system access.

Reply:
You still need to ensure it exists.